lib/mix_test_watch/command.ex

```diff
@@ -40,8 +40,10 @@ defmodule MixTestWatch.Command do
     Application.get_env(:mix_test_watch, :prefix, @default_prefix)
   end
 
-  defp task_command(task) do
-    ~s(MIX_ENV=test #{prefix} do #{ansi}, #{task})
+  defp task_command(task, args) do
+    ["MIX_ENV=test", prefix, "do", ansi <> ",", task, args]
```

Comment:
Was the most pretty way I found to be able to remove the commandline args if they are empty.